controllers/drcluster_controller.go

```diff
@@ -141,6 +145,24 @@ func (r *DRClusterReconciler) drClusterConfigMapMapFunc(configMap client.Object)
 	return requests
 }
 
+func (r *DRClusterReconciler) drClusterSecretMapFunc(secret client.Object) []reconcile.Request {
+	if secret.GetNamespace() != NamespaceName() {
+		return []reconcile.Request{}
+	}
+
+	drcusters := &ramen.DRClusterList{}
```

Comment:
We don't need to create the DRClusterList on the heap - we can do:

    drclusters := ramen.DRClusterList{}
    r.Client.List(context.TODO(), &drcusters)

Reply:
You are probably right, but it's also for consistency. We are always creating lists on heap (I've looked through all our code).